src/server/call.ts

```diff
@@ -121,6 +133,15 @@ export class Call {
 
   private callEventHandler(eventId: string, isBin: boolean = false) {
     return (data: any) => {
+      if (eventId === 'error') {
+        // An error's message is not enumerable, so doesn't get included
+        // by JSON.stringify(). Copy error info to a new, serializable object
+        let enumerableError: any = {};
+        Object.getOwnPropertyNames(data).forEach(key => {
+          enumerableError[key] = data[key];
+        });
+        data = enumerableError;
+      }
```

Comment:
This is the error passing fix i submitted. I think it's showing up here because you based your changes off the `serialize-error-message` branch, but you're submitting the pull against the `fix-stream-request` branch, which doesn't have that change